### PR TITLE
update the installation process

### LINKED ISSUE #147: Remove the rebuild semantic ui assets from readme/improve documentation
it is not necessary to get set up with the project and confusing for new contributors

https://github.com/AgileVentures/WebsiteOne-FE#rebuild-the-semantic-ui-asserts

### PR DESCRIPTION
**Pull request by abdellani**
#### Issue addressed

fixes #147

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -28,23 +32,6 @@ Run tests
 $ yarn test
 ```
 
-# Rebuild the semantic UI asserts
```
Comment (joaopapereira) [inline README.MD:31]: We are still using semantic UI, so I think we should keep this instructions here. Maybe add a disclaimer that this is only needed if we need to upgrade semantic

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Can you please review my comment

**Comment by abdellani** [comment]
@joaopapereira 
I updated README.md as you suggested.

**Comment by mattwr18** [comment]
thanks @abdellani, sorry I didn't think about that use case @joaopapereira brought up

**Review by joaopapereira — APPROVED** [review]
Looks good, going to merge

**Pull request merged**